### ISSUE TITLE
Child Hyper Parameters

### ISSUE THREAD

**Issue by zswartz1993** [melodyguan/enas#32]
Hi, could you briefly explain the meaning of:
"child_out_filters", "child_num_branches", "child_num_cell_layers", and "child_keep_prob"

I am fairly new to ML and am trying to figure out how to properly train the outputted fixed architectures.

Thank you so much! This really is a wonderful tool.

**Comment by karandwivedi42** [comment]
"child_out_filters" -> Number of channels of the convolution layers

 "child_num_branches", -> Number of options in ENAS (here it is fixed at 5)

 "child_num_cell_layers" -> Can't find this in the code..

"child_keep_prob" -> dropout keep probability [more here](https://www.tensorflow.org/api_docs/python/tf/nn/dropout)
